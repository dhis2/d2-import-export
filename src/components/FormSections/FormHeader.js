import React from 'react'
import styles from './FormHeader.module.css'

export const FormHeader = ({ icon, label }) => (
    <div className={styles.container}>
        <span className={styles.icon}>{icon}</span>
        <span className={styles.label}>{label}</span>
    </div>
)
