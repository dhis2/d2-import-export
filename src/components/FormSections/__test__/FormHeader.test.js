import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import { FormHeader } from '../FormHeader'

describe('Input component - FormHeader', () => {
    it('should render correctly', () => {
        const file = shallow(
            <FormHeader>
                <span>Test</span>
                Foobar
            </FormHeader>
        )

        expect(toJson(file)).toMatchSnapshot()
    })
})
