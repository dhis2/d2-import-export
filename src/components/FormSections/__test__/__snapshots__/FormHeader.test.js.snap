// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component - FormHeader should render correctly 1`] = `
<div
  className="container"
>
  <span
    className="icon"
  />
  <span
    className="label"
  />
</div>
`;
